src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -42,19 +53,17 @@ private void buildFTSQuery(CriteriaBuilder builder, CriteriaQuery query, Root ro
         CriteriaQuery<Monument> query = this.createCriteriaQuery(builder, false);
         Root<Monument> root = this.createRoot(query);
         root.fetch("tags", JoinType.LEFT);
-        query.select(root).distinct(true);
```

Comment:
We can't use distinct because we're ordering by the functions which are not in the SELECT clause. That being said, I haven't noticed any duplicate results yet so I'm tentatively hoping that it isn't necessary anyway for this query